src/platform/forms/tests/save-in-progress/SaveFormLink.unit.spec.jsx

```diff
@@ -52,18 +52,6 @@ describe('Schemaform <SaveFormLink>', () => {
 
     expect(tree.text()).to.contain('Finish this application later');
   });
-  it('should render expired message when not logged in and noAuth status', () => {
```

Comment:
Kept this and moved below but for logged in state. I think it can cover the case where the session expires while the user is logged in.